FreeRTOS/FreeRTOS
# Update the readme files to be correct for the upcoming LTS snapshot z…

Description
-----------
Update readme files ready for imminent LTS snapshot release.

Test Steps
-----------
N/A no code changes.

Related Issue
-----------
N/A no code changes.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
